src/components/Graph/MainGraph.js

```diff
@@ -443,7 +444,7 @@ class MainGraph extends Component {
                             height={80}
                             series={this.state.allDatesSeries}
                             dates={this.state.dates}
-                            x={margin.yAxis}
+                            x={margin.yAxis + (this.props.width * 0.03)}
```

Comment:
How did you derive this `0.03` number? If it's constant, could we move it to the `constants.js` file with a comment on how you calculated this value?

Reply:
Sure thing. The issue of why the brush area wasn't lining up with the graph using just margin.yAxis has to do with the .graph-wrapper div being set to flex, and the y-axis label being rendered from a div not svg. It is difficult to calculate how much space will be allotted for this before rendering graph, so hard to pass in to the component before Graph itself mounts. The 0.03 constant was derived from a few tests at different screen widths.